src/pages/home/report/ReportActionItemFragment.tsx

```diff
@@ -133,14 +133,7 @@ function ReportActionItemFragment({
             }
 
             if (isHoldReportAction) {
```

Comment:
@BartoszGrajdek coming from https://expensify.slack.com/archives/C049HHMV9SM/p1708470892526659?thread_ts=1708396751.518539&cid=C049HHMV9SM, what do you think of passing a prom from `ReportActionItemMessage` `isSystemMessage` and apply the muted style accordingly? I think it'd be nice as we don't have repeated code.

Reply:
I think it would be clean if we added this as a helper method in `ReportActionsUtils` to centralize this logic.

Reply:
Yeah I agree, that's why it all seemed confusing to me - there are several places in the app where we are setting muted color for system messages

Reply:
i.e. these [action types](https://github.com/Expensify/App/blob/c2f8a534b1bf1f21cf3b84162f051f3de2f97a29/src/pages/home/report/ReportActionItem.js#L463-L468) also have colorMuted style added